### PR TITLE
add example for context.site

### PR DESCRIPTION
**Pull request by Skn0tt**
This PR adds an example page for `context.site`.

### CODE REVIEW

**Comment by whitep4nth3r** [comment]
Hey @Skn0tt I'm on PTO this week. I can get to it next week, or please request a review from @philhawksworth ☺️

**Comment by Skn0tt** [comment]
ping

File: pages/context-site/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import repoLink from "../../components/repo-link.js";
+
+export default {
+  title: "Deleting cookies",
```
Comment (whitep4nth3r) [inline pages/context-site/index.js:4]: Bad copy paste! :P

File: pages/home/index.js [hunk 2]
```diff
@@ -68,6 +68,7 @@ export default {
         <li class="home__sectionListItem"><a class="home__sectionListItemLink" href="/example/log">Write to the logs</a></li>
         <li class="home__sectionListItem"><a class="home__sectionListItemLink" href="/example/environment">Use environment variables</a></li>
         <li class="home__sectionListItem"><a class="home__sectionListItemLink" href="/example/uncaught-exceptions">Read logs for uncaught exceptions</a></li>
+        <li class="home__sectionListItem"><a class="home__sectionListItemLink" href="/example/context-site">Access Site Information</a></li>
```
Comment (whitep4nth3r) [inline pages/home/index.js:71]: In keeping with the rest of the conventions, it's probably best to only uppercase "Access" here

File: pages/context-site/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      <p>Netlify Edge Functions give access to site information via <code>context.site</code>.</p>
+      <pre><code>import type { Context } from "netlify:edge";
+
+export default async (request: Request, context: Context) => {
+  return new Response(\`Hello from \${context.site.name}!\`);
+};</code></pre>
+
+      <h2>See this in action</h2>
+      <ul>
+        <li><a href="/context-site">See Response from code snippet</a></li>
```
[the remark this answers is not in the document]
Comment (Skn0tt) [reply pages/context-site/index.js:19]:
```suggestion
        <li><a href="/context-site">See response from code snippet</a></li>
```

File: pages/home/index.js (hunk 2 above)
Comment (Skn0tt) [reply pages/home/index.js:71]:
```suggestion
        <li class="home__sectionListItem"><a class="home__sectionListItemLink" href="/example/context-site">Access site information</a></li>
```

File: pages/context-site/index.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import repoLink from "../../components/repo-link.js";
+
+export default {
+  title: "Deleting cookies",
+  metaDescription: "Delete an HTTP cookie",
```
Comment (Skn0tt) [inline pages/context-site/index.js:5]:
```suggestion
  metaDescription: "Access Site Information from an Edge Function",
```

**Review by whitep4nth3r — APPROVED** [review]

**Pull request merged**